01-Git_Bash_Make/guess_a_number.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+echo "Enter number from 0 to 9: "
+read userinput
+
+if [[ $userinput -lt 0 || $userinput -gt 9 ]]
+then
+	echo "Input is out of range!"
+	exit -1
+fi
+
+gen_value=$(( ( RANDOM % 10 ) ))
+echo $gen_value
+
+if [[ gen_value -eq userinput  ]]
+then
+  echo "You Win! Congratulations."
+else
+  echo "You lose... Try again."
+fi
```

Comment:
add empty line at the end of file
```suggestion
fi

```